packages/starlight-blog/package.json

```diff
@@ -23,13 +23,13 @@
     "github-slugger": "2.0.0"
   },
   "devDependencies": {
-    "@astrojs/starlight": "0.11.0",
+    "@astrojs/starlight": "^0.11.0",
     "@playwright/test": "1.36.1",
-    "astro": "3.2.3"
+    "astro": "^4"
   },
   "peerDependencies": {
     "@astrojs/starlight": ">=0.11.0",
-    "astro": ">=3.2.0"
+    "astro": "3-4"
```

Comment:
I would rather test the code with astro 5 and beyond before allowing users to use it. Of course one has to keep an eye on this but really strange and annoying things can happen for the users otherwise.